docs/mana_proposal.md

```diff
@@ -0,0 +1,100 @@
+# Mana Implementation Proposal
+
+The goal of this document is to provide a high level overview of how mana will be implemented in GoShimmer.
+
+## Introduction
+
+Mana is a a reputation system for nodes within the IOTA network.
+
+Reputation is gained by contributing to the network, i.e. creating value transfers.
+Reputation is lost when not cooperating with the network: using up all the available bandwidth, advocating for
+"bad" transfers during voting, etc..
+
+As time passes, earned mana of a node decays to encourage keeping up the good behavior.
```

Comment:
"earned mana of a node decays to encourage keeping up the good behaviour" - this is only true for base mana 2, base mana 1 does not decay. So maybe this sentence can be misleading to some people.

Reply:
good point, I will update it.